BossMod/Modules/Shadowbringers/Foray/Duel/Duel2Lyon/Duel2Lyon.cs

```diff
@@ -0,0 +1,30 @@
+namespace BossMod.Shadowbringers.Foray.Duel.Duel2Lyon
+{
+
+class Lyon2DuelStates : StateMachineBuilder
```

Comment:
Looks like something is off with your formatting...

Reply:
ok I changed the formating